app/_how-tos/create-a-key-set-with-a-jwk.md

```diff
@@ -24,7 +24,24 @@ min_version:
 
 tldr:
   q: How do I create a JWK and add it to a Key Set?
-  a: Create a Key Set with the `/key-sets` endpoint, then create a Key and configure the `set.id` or `set.name` parameter to point to the Key Set. 
+  a: Create a Key Set with the `/key-sets` endpoint, then create a Key and configure the `set.id` or `set.name` parameter to point to the Key Set.
+
+prereqs:
+  inline:
+    - title: JSON Web Key
+      content: |
+        This tutorial requires a JSON Web Key. You can generate your own or use this one:
```

Comment:
```suggestion
        This tutorial requires a JSON Web Key. You can generate your own or use this one for testing:
```
Let's make sure that we call out that this is for testing so we don't have someone coming back to us about it being insecure.